src/typescript.gen.spec.ts

```diff
@@ -202,17 +202,17 @@ async all_datagroups(
     it('encodes string path params', () => {
       const method = apiTestModel.methods['run_url_encoded_query']
       const expected =
-`model_name = this.encodePathParam(model_name)
-view_name = this.encodePathParam(view_name)
-result_format = this.encodePathParam(result_format)
+`model_name = encodeParam(model_name)
+view_name = encodeParam(view_name)
+result_format = encodeParam(result_format)
 `
       const actual = gen.encodePathParams('', method)
       expect(actual).toEqual(expected)
     })
-    it('encodes only string path params', () => {
+    it('encodes only string or date path params', () => {
```

Comment:
hm, the name of the test is misleading. `run_look` has `look_id` (int) and `result_format` (string) so maybe

```suggestion
    it('encodes does not encode int path params', () => {
```

or at least a comment to the effect of
```suggestion
    // TODO eventually add method that has a date type path param
    it('encodes only string or date path params', () => {
```